src/components/EventDetails/EventDetails.js

```diff
@@ -877,27 +872,31 @@ export class EventDetail extends React.Component<Props> {
         eventData = {
           actionTitle: t('label.activated'),
           actionSubtitle: this.getFeeLabel(event),
-          buttons: isPPNActivated ? [referFriendsButton] : [activatePillarNetworkButton, referFriendsButtonSecondary],
+          buttons: isPPNActivated || !isArchanovaAccountActive
+            ? [referFriendsButton]
+            : [activatePillarNetworkButton, referFriendsButtonSecondary],
         };
         break;
       case PAYMENT_NETWORK_ACCOUNT_TOPUP:
-        const topUpMoreButton = {
-          title: t('button.topUpMore'),
-          onPress: this.topUpPillarNetwork,
-          secondary: true,
-        };
-        eventData = {
-          buttons: isPending
-            ? [topUpMoreButton]
-            : [
-              {
-                title: t('button.send'),
-                onPress: this.sendSynthetic,
-                secondary: true,
-              },
-              topUpMoreButton,
-            ],
-        };
+        if (isArchanovaAccountActive) {
```

Comment:
Consider inverting the if for readability
```suggestion
        if (!isArchanovaAccountActive) break;
```